docs/advanced-topics/advanced-methods.md

```diff
@@ -0,0 +1,100 @@
+(advanced-methods)=
+# {octicon}`mortar-board` Advanced Methods
+Many of the models supported by finetuner make use of similar methods throughout finetuning, such as the methods used for *calculating loss*, *sampling* and *pooling* . Finetuner offers alternatives to each of these methods, which can improve the performance of your finetuning run in some cases.
+
+## Loss Functions
+
+Loss functions are used to calculate the quality of embeddings while training; the higher the output of the loss function, the more the parameters of the model will be updated.
+By default we use Triplet Margin Loss, however we support many other loss function as well, including ArcFace Loss and CosFace Loss.
+
+```{Important}
+Please check the [developer reference](../../api/finetuner/#finetuner.fit) to get the available options for `loss`.
+```
+
+### Triplet Margin Loss
+
+Lets first take a look at our default loss function, Triplet Margin Loss.  
+
+Triplet Margin Loss is a *contrastive* loss function, meaning that the loss is calculated by comparing the embeddings of multiple documents to each other, three documents, to be exact.
+Each triplet of documents consists of an anchor document, a positive document and a negative document.
+The anchor and the positive document belong to the same class, and the negative document belongs to a different class.
+The goal of Triplet margin loss is to maximise the difference between the distance from the anchor to the positive document, and the distance from the anchor to the negative document.
+For a more detailed explanation on Triplet Loss, as well as how samples are gathered to create these Triplets, see {doc}`/advanced_topics/negative-mining`.
+
+### SphereFace losses
+
+SphereFace loss is a loss function that was first forumalted for computer vision and face recognition tasks,
```

Comment:
let's make sure if we always use `SphereFace loss` or `SphereFaceLoss` or `SphereFace Loss` (see above)

Reply:
Since SphereFace isnt a supported loss function like the others, ill keep it to 'SphereFace loss'